include/mbedtls/ssl.h

```diff
@@ -84,8 +84,13 @@
 #define MBEDTLS_ERR_SSL_BAD_CERTIFICATE                   -0x7A00
 /* Error space gap */
 /**
```

Comment:
We need a two changelog entries:

* Bug fix: some connections that worked in 3.5 no longer work in 3.6.0.
* Behavior change: applications that were relying on the semi-documented behavior of `MBEDTLS_ERR_SSL_RECEIVED_NEW_SESSION_TICKET` now need to call a function to enable this behavior.